dss/util/async_state.py

```diff
@@ -21,16 +21,23 @@ class MyAsyncSubclass(AsyncStateItem):
 
     def __init__(self, key: str, body: dict) -> None:
         self.key = key
+        body = body.copy()
         if not body.get('_type'):
             body['_type'] = type(self).__name__
         self.body = body
 
     def _put(self) -> typing.Any:
         return put_item(table=self.table, value=json.dumps(self.body), hash_key=self.key)
 
+    @property
+    def data(self):
+        data = self.body.copy()
+        del data['_type']
```

Comment:
dumb q: why the deletion operation here?

Reply:
To make this assertion pass:
```
in_data = dict(my_key=foo)
AsyncStateItem.put(key, in_data)
assert in_data == AsyncStateItem.get(key).data
```